damus/ContentView.swift

```diff
@@ -774,8 +775,21 @@ func setup_notifications() {
             return
         }
     }
+
 }
 
+func setup_notifications_permission() {
```

Comment:
not sure what the point of this is, isn't this already handled by `setup_notifications` ?